docs/source/esmeralda.rst

```diff
@@ -1,5 +1,277 @@
+
+.. image:: images/esmeralda/maps_temporal_evolution.png
+   :width: 900
+   :align: center
+
+If the ``apply_temp`` parameter is *True*, these variations will be considered for the correction. On the contrary, this variable must be set to *False* in case the map does not include this temporal information table, or the city is run over MC files (whose events do not have a timestamp variable either).
+
+
+The following image shows the clear improvement on the energy spectrum after applying all the corrections explained above:  
+
+.. image:: images/esmeralda/energy_spectrum_corr_vs_uncorr.png
+   :width: 900
+   :align: center
+
+
+
+
+
+|
+|
+
+.. warning::
+ Despite the energy correction explained just before, there are **two** important factors that make the previous calibration not ultimate. They are not going to be explained in detail here, seeing that these further corrections are not applied along `Esmeralda`. However, since they are not applied inside any other city either and the energy modification of events is performed here, it is justified to comment them now.              
+
+1. **Non-linearities at high energies**. Due to the significant difference between the krypton energy scale and the one of the physics data (above 1 MeV), the Kr-based energy correction might **not be sufficient** for all the energy range considered. Therefore, although krypton maps were applied, it is advisable to check the high energy peaks, so as to account for observed **non-linearities** and obtain the proper calibration. The plots below show clearly how the high energy 208-Thallium gamma lines (nominal values are illustrated with dashed red lines) are not aligned perfectly in spite of the maps corrections.
+
+.. image:: images/esmeralda/energy_spectrum_corr_vs_uncorr_PEAKS.png
+   :width: 900
+   :align: center
+
+*Note:* the energy scale of the uncorrected hits coming from the hdst presented in the plots above has only illustrative pruposes (to compare both distributions). Its conversion between pes to MeV was made considering in an approximate way the number of pes yielded by krypton events.
+
+
+2. **The axial length (Z-width) effect**. There is an additional final energy correction that must be applied in the analysis post-reconstruction: the so-colled *Z-width effect* correction. Detailed information about possible explanations for this phenomenon, as well as an empirical procedure to deal with it can be found `here <https://arxiv.org/abs/1905.13110>`_.
```

Comment:
so-called

the link may be to the actual published paper and not to the axiv, maybe to the inspirehep entry?